### PR TITLE
[PR] Account for some default Content Syndicate attributes 

### LINKED ISSUE #60: Offset attribute
Offset attribute is not functioning.

### PR DESCRIPTION
**Pull request by philcable**
Fixes #60 and #61...

### CODE REVIEW

**Comment by philcable** [comment]
@jeremyfelt #reviewmerge?

File: includes/university-center-syndicate-shortcode-organization.php [hunk 1]
```diff
@@ -116,18 +127,62 @@ public function display_shortcode( $atts ) {
 	 * @return string The generated HTML for an individual organization.
 	 */
 	private function generate_item_html( $organization, $type ) {
-		if ( 'headlines' === $type ) {
```
Comment (jeremyfelt) [inline includes/university-center-syndicate-shortcode-organization.php]: It looks like this removes support for the "headlines" output type. Is that intentional?

File: includes/university-center-syndicate-shortcode-person.php [hunk 2]
```diff
@@ -116,18 +127,62 @@ public function display_shortcode( $atts ) {
 	 * @return string The generated HTML for an individual person.
 	 */
 	private function generate_item_html( $person, $type ) {
-		if ( 'headlines' === $type ) {
```
Comment (jeremyfelt) [inline includes/university-center-syndicate-shortcode-person.php]: Same question about "headlines" support here.

File: includes/university-center-syndicate-shortcode-project.php [hunk 3]
```diff
@@ -116,18 +127,62 @@ public function display_shortcode( $atts ) {
 	 * @return string The generated HTML for an individual project.
 	 */
 	private function generate_item_html( $project, $type ) {
-		if ( 'headlines' === $type ) {
```
Comment (jeremyfelt) [inline includes/university-center-syndicate-shortcode-project.php]: And here :)

File: includes/university-center-syndicate-shortcode-publication.php [hunk 4]
```diff
@@ -116,18 +127,62 @@ public function display_shortcode( $atts ) {
 	 * @return string The generated HTML for an individual publication.
 	 */
 	private function generate_item_html( $publication, $type ) {
-		if ( 'headlines' === $type ) {
```
Comment (jeremyfelt) [inline includes/university-center-syndicate-shortcode-publication.php]: And also here. :) (sorry for the duplication)

File: includes/university-center-syndicate-shortcode-organization.php (hunk 1 above)
Comment (philcable) [inline includes/university-center-syndicate-shortcode-organization.php]: It wasn't intentional :)

My thinking was that the title markup should/will display regardless of output type.

Comment (philcable) [inline includes/university-center-syndicate-shortcode-organization.php]: As an addendum to my above comment (I realize it's quite vague :) - this approach is working as intended, but I'm happy to adjust it if needed!

Comment (jeremyfelt) [inline includes/university-center-syndicate-shortcode-organization.php]: Ahhh, I get it now. I'm guilty of looking at the diff and not the full picture. :+1:

**Comment by philcable** [comment]
@jeremyfelt Let me know if my reasoning and/or response makes sense :) Thanks!

**Pull request merged**